pkg/util/template/resource.go

```diff
@@ -25,6 +25,9 @@ type HTML struct {
 	Name string
 	// ComponentDependencies is the HTML component templates this template depends on.
 	ComponentDependencies []*HTML
+	// IsFindAllowedInFs returns an boolean indicating if the Fs can be used to find this resources
+	// For example, if you don't want a html inside the app fs to be used, it should return false.
+	IsFindAllowedInFs FsFilter
```

Comment:
FindResourcesFsFilter is a better name. What do you think?

Reply:
Renamed to FindResourcesFsFilter